### PR TITLE
Default the number of YSQL shards/tserver to 2

### PR DESCRIPTION
**Pull request by WesleyW**
Default number of YSQL shards/tserver to 2. 
https://github.com/YugaByte/yugabyte-db/issues/2137

### CODE REVIEW

**Review by bmatican — COMMENTED** [review]
Can you check if we need to store this in the cluster config file?

**Comment by bmatican** [comment]
Just checked and we don't save this to the config. However, let's fix the trafis build by updating the version to download, then we're good to go!

File: bin/yb-ctl [hunk 1]
```diff
@@ -368,7 +368,7 @@ class Installer:
                 logging.error(
                     "Directory %s exists but does not appear to be a valid YugaByte DB "
                     "installation directory. Remove that directory and re-run the script "
-                    "to re-install YugaByte DB.", installation_dir)
+                    "to re-install YugaByte DB.", self.installation_dir)
```
Comment (bmatican) [inline bin/yb-ctl:371]: oops :)

**Review by bmatican — APPROVED** [review]
Probably unrelated to this, but one of the Travis tests actually still fails, but at least now it picks up 1.3.2.1 correctly.

**Comment by WesleyW** [comment]
The failed Travis test is one @ndeodhar is working on, so I've ignored it.

**Pull request merged**